src/source-updater.js

```diff
@@ -43,6 +45,7 @@ export default class SourceUpdater {
     this.pendingCallback_ = null;
     this.timestampOffset_ = 0;
     this.mediaSource = mediaSource;
+    this.firstAppend_ = false;
```

Comment:
I feel this name is a bit ambiguous. Not sure if it's worth fixing but something like `processedAppend_` might be more clear.